neurodsp/plts/time_series.py

```diff
@@ -57,11 +57,15 @@ def plot_time_series(times, sigs, labels=None, colors=None, ax=None, **kwargs):
 
     if colors is not None:
         colors = repeat(colors) if not isinstance(colors, list) else cycle(colors)
-    else:
-        colors = cycle(['k', 'r', 'b', 'g', 'm', 'c'])
+    elif len(sigs) <= 2:
+        colors = cycle(['k', 'r'])
+
+    for time, sig, label in zip(times, sigs, labels):
 
-    for time, sig, color, label in zip(times, sigs, colors, labels):
-        ax.plot(time, sig, color, label=label)
+        if colors is not None:
```

Comment:
Ahh, I see, because we use zip, we can't just leave the singular None. 

It's a little pedantic, but I would suggest updating above to:
```
# If not provided, default colours for up to two signal to black & red
if not colors and len(sigs) <= 2:
    colors = cycle(['k', 'r'])
colors = repeat(colors) if not isinstance(colors, list) else cycle(colors)
```

This way, a colors of None gets the repeat call, and we need less code overall to do this (we don't have to update the loop). Does that make sense?

Reply:
The issue with this is that ax.plot doesn't like None. This will raise an error:
```ax.plot(times, sig, None)```

Reply:
Oh, it does need the explicit keyword. It does some kind of "what is this" guess procedure for what `arg` inputs are (because it supports variable APIs), but `color=None` works, I think.

So, the plot call should be valid as:
`ax.plot(time, sig, color=color, label=label)`

Reply:
Using None as a kwarg works, updated!

Reply:
awesome!

So I direct committed a small tweak we don't need to double on calling cycle, or use the else, right?

Reply:
Yeah, that's better. It functions the same and saves a line.

Reply:
I went for one more small direct edit - editing back to the original & consistent zip approach.

Reply:
Tests pass! This looks good to me.